app/assets/javascripts/angular/components/myOrganizationsDashboardList/myOrganizationsDashboardList.html

```diff
@@ -6,4 +6,7 @@
             load-details="false"
             edit-mode="$ctrl.myOrganizationsDashboard.editOrganizations">
     </organization-overview>
+    <section class='tc' ng-if="$ctrl.rootOrgs.length <=0">
```

Comment:
I need to enable prettier on HTML now that it supports it. As long as it doesn't butcher the Angular template syntax...
```suggestion
    <section class='tc' ng-if="$ctrl.rootOrgs.length <= 0">
```

Reply:
thx, yah, usually im pretty good with those spaces on conditions, but it slipped!